src/include/iscsi-lio-server/helps.rb

```diff
@@ -129,10 +131,10 @@ def initialize_iscsi_lio_server_helps(include_target)
          ) +
           _(
             "<p>With <b>Edit LUN</b> one can modify the LUN mapping. Please note that LUN target number" +
-            " must be unique.<br>After pressing <b>Edit Auth</b>, select the type of authentication." +
-            " Use <b>Incoming</b>, <b>Outgoing</b> or both together. Then insert <b>User</b> and <b>Password</b>." +
-            " Please make sure to set different passwords for incoming and outgoing authentication.\n" +
-            " If <b>Use Authentication</b> is disabled in previous dialog, <b>Edit Auth</b> is disabled here.</p>"
+            " must be unique.<br>After pressing <b>Edit Auth</b>, it's needed to " +
+            " Use <b>Authentication by Targets</b> and  <b>Authentication by Initiators</b> together. Then insert <b>UserID / Password</b> and <b>UserID_Mutual / Password_Mutual</b>." +
```

Comment:
Um, I'm not familiar with the iSCSI terminology (and I have to admit it's a bit confusing to me), but maybe I'd put the `Incoming` and `Outgoing` terms in parenthesis to make it more understandable for less experienced users (like me :wink: ).